src/amo/components/SearchResult/index.js

```diff
@@ -59,10 +63,9 @@ export class SearchResultBase extends React.Component<InternalProps> {
     let imageURL = iconURL;
 
     if (addon && isTheme(addon.type)) {
-      // Since only newly created static themes will have more than one preview
-      // we will set up a fallback for now.
-      let themeURL =
-        getPreviewImage(addon, { index: 1 }) || getPreviewImage(addon);
+      const previewIndex = getPreviewIndexBySize(addon);
```

Comment:
This is confusing because `getPreviewIndexBySize()` looks for a default width of 720 but I can't look at this function and no it. Is there a better name? Maybe `getWidePreviewImageIndex()`?